templates/workflow/workflow.tpl

```diff
@@ -58,6 +58,14 @@
 			>
 				{translate key="common.preview"}
 			</pkp-button>
+			{if $submissionPaymentsEnabled}
+				<pkp-button
+					ref="submissionPaymentsButton"
+					@click="openSubmissionPayments"
+				>
+					{translate key="common.payments"}
+				</pkp-button>
+			{/if}
```

Comment:
Because this is a small, one-field form. I recommend using a dropdown instead of a modal. This is how the relations form is handled in OPS: https://github.com/pkp/ops/blob/master/templates/workflow/workflow.tpl#L125-L130

Reply:
Done!